drivers/scheduler/scheduler.go

```diff
@@ -69,6 +69,9 @@ type Driver interface {
 
 	// DeleteVolumes will delete a storage volume.
 	DeleteVolumes(*Context) error
+
+	// GenerateSupportBundle generates a bundle that can be used by support - logs, cores, states, etc
+	GenerateSupportBundle(*Context) (string, error)
```

Comment:
But Describe() is a k8s specific term which doesn't imply to anyone that possibly a log file will be generated and dumped somewhere. How will that make it look generic?

Reply:
The above function is not generating a log file. It is the caller of the function who uses the output to create the log file. The above function is serving the purpose of getting all details of the given context.

Yes, Describe() is a k8s operation but can be extended generally too since it aptly represents the purpose of the function.